dmsite/venv/src/Datamaster.js

```diff
@@ -1,53 +1,27 @@
 import React, { Component } from 'react';
+import { Layout } from 'antd';
 import 'filepond/dist/filepond.min.css';
-import { Switch, Route } from 'react-router-dom';
 import Header from './Header/Header';
-import Home from './HomeComponent';
-import DatasetDetails from './DatasetDetails/DatasetDetails';
-import Login from './Login';
-import SearchResultsPage from './SearchResultsPage/SearchResultsPage';
+import {withRouter} from "react-router-dom";
 
 
+const { Content, Footer } = Layout;
 class DataMaster extends Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            data: null,
-            error: ''
-        }
-    }
-
-//    componentDidMount() {
-//        this.getData();
-//    }
-//
-//    getDAta() {
-//        this.setSTate({ data: data });
-//    }
-
-//    _filterCampaign(query) {
-//       fetch(`someendpoint/?query=${query}`)
-//        .then((json) => {
-//           this.setState({ data: json.data });
-//        })
-//        .catch((error) => {
-//            this.setState({ error });
-//        })
-//    }
-
     render() {
         return (
-            <main>
+            <Layout className='Layout'>
                 <Header />
-                <Switch>
-                    <Route path = '/search' component={SearchResultsPage} />
-                    <Route path = '/details' component={DatasetDetails} />
-                    <Route path = '/home' component={Home} />
-                    <Route path = '/' component={Login}/>
-                </Switch>
-            </main>
+                <Content style={{padding: '0 50px'}}>
```

Comment:
You can using `paddingRight: 50px` here if you just want to just specify the right padding.